### PR TITLE
Test driver

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex [hunk 1]
```diff
@@ -317,15 +317,15 @@ \section{Tests}
...
+its target, but a \texttt{.sha512} extension is added, and the \texttt{.sha512}
+file's contents are only the SHA512 hash of its target. Content links for data
 files in a Git distributed version control repository prevent repository
-bloat. To obtain content links, register an account with the \texttt{ITK}
-community at \url{https://midas3.kitware.com} and request upload permissions
-on the ITK discussion. The content links may also be created locally: once
-the actual file at issue has been placed in the corresponding directory, run
-CMake configuration, and CMake provides the desired \texttt{.md5} extension
-content link file.
+bloat. To obtain content links, register an account community at
```
Comment (jhlegarreta) [inline SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex]: Just wondering if the wording "register an account community" is OK.

File: SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex [hunk 2]
```diff
@@ -613,10 +613,10 @@ \subsection{Naming Files}
 \subsubsection{Naming Tests}
 \label{subsubsec:NamingTests}
 
-Following the \code{TestDriver} philosophy, test files must be named with
-the same name used to name the \code{main} method contained in the test file
-(\code{.cxx}). This name should generally be indicative of the class tested,
-e.g.
+Following the \code{TestDriver} philosophy, Section~\ref{sec:Tests}, test
```
Comment (jhlegarreta) [inline SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: AFAIK, in the ITK SW Guide it is customary to add the **pageref** as well e.g. `Section~\ref{sec:Tests} on page~\pageref{sec:Tests}` for the sake of completion

File: SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex (hunk 1 above)
Comment (thewtex) [reply SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex]: Good catch. Removing `community`...

File: SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex (hunk 2 above)
Comment (thewtex) [reply SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: Adding.

**Pull request merged**